raw/map.go

```diff
@@ -13,6 +14,25 @@ import (
 // StringSliceMap represents an array of strings or a map of strings.
 type StringSliceMap map[string]string
 
+// Value returns the map in JSON format
```

Comment:
Hate to be nit-picky but could we get `.` to follow this comment?

```suggestion
// Value returns the map in JSON format.
```